src/landscape/components/LandscapeForm/BoundaryStep.js

```diff
@@ -367,11 +367,13 @@ const getOptionComponent = option => {
 };
 
 const BoundaryStep = props => {
-  const [option, setOption] = useState(OPTION_SELECT_OPTIONS);
   const [boundingBox, setBoundingBox] = useState();
   const isMounted = useIsMounted();
-  const OptionComponent = getOptionComponent(option);
   const { landscape, onSave, setUpdatedLandscape } = props;
+  const [option, setOption] = useState(
+    landscape.boundaryOption || OPTION_SELECT_OPTIONS
```

Comment:
question:  what does OPTION_SELECT_OPTIONS mean? That name is confusing.

Reply:
@paulschreiber That is referencing the options screen where all the available options for boundary are displayed, maybe it can be `OPTION_DISPLAY_OPTIONS`? or can you think of a better name?

Reply:
Maybe something like`OPTION_BOUNDARY_MENU` or `OPTION_BOUNDARY_CHOICES` if that is the intent.

Reply:
I like `OPTION_BOUNDARY_CHOICES`